modules/parent-join/src/test/java/org/elasticsearch/join/aggregations/ChildrenToParentAggregatorTests.java

```diff
@@ -90,7 +90,7 @@ public void testParentChild() throws IOException {
             new ShardId(new Index("foo", "_na_"), 1)
         );
         // TODO set "maybeWrap" to true for IndexSearcher once #23338 is resolved
```

Comment:
maybe also remove TODOs like this one? (regarding wrapping the reader)

Reply:
done